layers/+lang/bibtex/packages.el

```diff
@@ -110,3 +112,48 @@
         "ld" 'doi-utils-add-bibtex-entry-from-doi
         "li" 'isbn-to-bibtex
         "lp" 'pubmed-insert-bibtex-from-pmid))))
+
+(defun bibtex/init-ebib ()
+  (use-package ebib
+    :defer t
+    :init
+    (spacemacs/set-leader-keys "ate" 'ebib)
+    :config
+    (setq ebib-bibtex-dialect 'biblatex)
+
+    (evilified-state-evilify-map ebib-index-mode-map
+      :mode ebib-index-mode
+      :bindings
+      "j" 'ebib-next-entry
+      "k" 'ebib-prev-entry
+      "J" 'evil-scroll-page-down
+      "K" 'evil-scroll-page-up
+
+      "/" 'ebib-search
+      "n" 'ebib-search-next)
+
+    (spacemacs/set-leader-keys-for-major-mode 'ebib-index-mode
+      "j" 'ebib-jump-to-entry
+      "k" 'ebib-kill-entry
+      "b" 'biblio-lookup)
```

Comment:
This is from `biblio` package. So it should not be configured in `ebib`.

Reply:
But it is bound in the `ebib-index-mode` map, so if I would put it under biblio, we would have a similar argument. However, it really is ebib using biblio here as complementary package, not biblio using ebib(-index) as complementary package. So I really think it is better as it is (but other good arguments are welcome of course).

Reply:
You're right.